sdk/appcenter-crashes/src/test/java/com/microsoft/appcenter/crashes/CrashesTest.java

```diff
@@ -1608,6 +1572,7 @@ public ManagedErrorLog answer(InvocationOnMock invocation) {
     }
 
     @Test
+    @PrepareForTest({android.util.Log.class})
```

Comment:
import?

Reply:
I addressed static import of `android.util.Log.getStackTraceString` in other places, but I don't see how class import would work in this case. The fully qualified name `android.util.Log` needs to be used instead of `Log` to avoid namespace conflict with `com.microsoft.appcenter.ingestion.models.Log`. At the same time, I don't see using fully qualified name hurt readability (if that was your main concern), so it should be fine in this case.